testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_cbl_log_redaction.py

```diff
@@ -35,50 +37,51 @@ def test_mask_password_in_logs(params_from_base_test_setup, password):
     db = params_from_base_test_setup["db"]
     cbl_db = params_from_base_test_setup["source_db"]
     sync_gateway_version = params_from_base_test_setup["sync_gateway_version"]
-    log_file = params_from_base_test_setup["log_filename"]
+    log_filename = params_from_base_test_setup["log_filename"]
+    log_file = params_from_base_test_setup["test_db_log_file"]
     liteserv_platform = params_from_base_test_setup["liteserv_platform"]
+    testserver = params_from_base_test_setup["testserver"]
 
-    num_cbl_docs = 5
+    num_cbl_docs = 500
     # Cannot run on iOS as there is no support in xcode to grab cbl logs
-    if sync_gateway_version < "2.0.0" or liteserv_platform.lower() == "ios":
-        pytest.skip('This test cannnot run with sg version below 2.0 and if platform is iOS')
+    if sync_gateway_version < "2.0.0" and log_file is not None:
```

Comment:
skipping the test with log_file none applies only to iOS, it should work for other platforms, though file logging is not enabled. Can you change the condition?

Reply:
That's not true. Anytime, you runs the test without option  --enable-file-logging, it will remain initialised with None. Only if you pass the parameter it will receive the file name. I've made changes in conftest and I guess you missed them. Do check and let me know if you are ok.